src/lib.rs

```diff
@@ -2803,6 +2818,8 @@ impl Build {
                     format!("arm-kmc-eabi-{}", gnu)
                 } else if target.arch == "aarch64" && target.vendor == "kmc" {
                     format!("aarch64-kmc-elf-{}", gnu)
+                } else if target.os == "nto" {
+                    "qcc".into()
```

Comment:
It sounds like this target _only_ supports compiling with `qcc`, is that true?

If so, @NobodyXu is this the right place to set this up? This would only select the _default_ compiler, right? What if the user passes `cc::Build::new().compiler("clang")`? Is the expected behaviour to still try to use Clang, even though we know that'll fail?

Reply:
> It sounds like this target _only_ supports compiling with `qcc`, is that true?

I think/guess you have to use `qcc` if you want to include system header files due to license reasons. In other words, usually only `qcc` is used.
If someone wants to use a very non-default compiler, they can set it via environment variables.

Reply:
My concern is that if we add the `-V` flag, then you won't be able to compile with other compilers that don't support that flag

Reply:
I get your point. Thinking about my previous statement: It's not just header files, you also cannot link to create executables or shared objects without license violations. I don't think we need to support non-qcc compilers.

Reply:
Cool, thanks for the clarification.

Gonna keep this open pending answer wrt. default vs. required compiler.

Reply:
I don't think anyone is compiling for QNX Neutrino and not using `qcc`. If you pay this much for your RTOS, you colour between the lines, not outside.

Reply:
I meant, I wanted @NobodyXu to confirm that this was the best place to set it up ;)